docs/detections/rules-prebuilt.asciidoc

```diff
@@ -0,0 +1,143 @@
+[[rules-prebuilt]]
+== Install Elastic prebuilt rules
+
+:frontmatter-description: Start detections quickly with prebuilt rules designed and updated by Elastic.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [manage]
+
+Follow these guidelines to start using the {es-sec-app}'s <<prebuilt-rules, prebuilt rules>>, keep them updated, and make sure they have the data needed to run successfully. 
+
+* <<load-prebuilt-rules>>
+* <<prebuilt-rule-tags>>
+* <<select-all-prebuilt-rules>>
+* <<update-prebuilt-rules>>
+* <<download-prebuilt-rules>>
+* <<rule-prerequisites>>
+
+[NOTE]
+====
+* Apart from the Elastic Endpoint rule, prebuilt rules do not start running by default. You must first install the rules, then enable them.
```

Comment:
Slight tweak to Georgii's suggestion -- I feel like I've been encouraged in the past to avoid "out of the box" as somewhat colloquial:
```suggestion
* Prebuilt rules do not start running by default. You must first install the rules, then enable them. After installation, only a few prebuilt rules will be enabled by default, such as the Endpoint Security rule.
```